domains/dear.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "kirokir",
+    "email": "itsmelorika@gmail.com"
+  },
+  "records": {
+    "CNAME": "dearnco-django.onrender.com"
+  },
+  "proxied": true
```

you can't proxied onrender
```suggestion
  }
```